addons/bestja_frontend_fpbz/views/pages.xml

```diff
@@ -0,0 +1,79 @@
+                                <img id="BZ_logo" src="/bestja_frontend_fpbz/static/src/img/BZ_logo.png"/>
+                                <ul id="nav_menu_top" class="navbar navbar-right">
+                                    <li>
+                                        <a>Biuletyn wolontariuszy</a>
+                                        <a>Zostań partnerem</a>
+                                        <a>Zaloguj się</a>
+                                    </li>
+                                </ul>
+                                <div id="main_top_white_text">
+                                    <h1>Rocznie w Polsce marnuje się 9 mln ton jedzenia</h1>
+                                    <h3>2,5 mln osób nie ma dostępu do pełnowartościowej żywności</h3>
+                                    <h2>Pomóż nam to zmienić!</h2>
+                                </div>
+                                <div class="text-center">
+                                    <button id="choose_place" class="btn btn-default btn-lg text-center">wybierz swoją miejscowość</button><br />
+                                </div>
+                                <div class="col-md-4 col-md-offset-4 text-center centered">
+                                    <img class="img mouse_img" src="/bestja_frontend_fpbz/static/src/img/myszka.png"/>
+                                    <br />
+                                    <span id="direction_down">&lt;</span>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                </div>
+            </xpath>
+        </template>
+        <template id="new_footer" inherit_id="website.layout" customize_show="True" name="New Footer">
+            <xpath expr="//footer" position="replace">
+                <div class="container-fluid footer_div">
+                    <div class="row clearfix">
+                        <div class="col-footer text-center">
+                            <ul id="footer_icons">
+                                <li>
+                                    <a class="facebook" href="http://facebook.com"></a>
+                                </li>
+                                <li>
+                                    <a class="youtube" href="http://youtube.com"></a>
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="col-footer text-center">
+                            <h4><a href="http://www.wolontariat.bankizywnosci.pl/">O nas</a>
+                                <a href="http://www.wolontariat.bankizywnosci.pl/">Kontakt</a></h4>
+                        </div>
+                        <div class="col-footer text-right copyrights">
+                            <div>
+                                <p style="color: #FF9900">© 2014, Federacja Banków Żywności</p>
```

Comment:
Zamiast "Federacja Banków Żywności" `<span t-field="res_company.name">`. Zostanie wtedy wyświetlona taka nazwa organizacji jaka jest skonfigurowana w systemie. Tak jest też w domyślnej skórce Odoo i to jest sensowne, bo dzięki temu można bardzo szybko wszędzie zmienić tę nazwę, modyfikując tylko konfigurację w panelu.